osu.Game.Rulesets.Mania/Edit/Blueprints/ManiaPlacementBlueprint.cs

```diff
@@ -56,7 +56,7 @@ protected override bool OnMouseDown(MouseDownEvent e)
 
         protected override void OnMouseUp(MouseUpEvent e)
         {
-            EndPlacement();
+            EndPlacement(true);
```

Comment:
As a side note this doesn't necessarily match the behaviour in the standard editor as `BeginPlacement` is called in `OnMouseDown` here (so it is only then that the placement is shown in the mania editor, instead of on mouse movement). Out of scope here, since at a quick glance it'd require some higher-level placement logic changes, and you probably know that, but I figured I'd say it just in case.

Reply:
the mania editor right now needs a pass to fix its behaviour (aiming to get osu! mode editor perfected first).

as such we haven't tested mania at all yet, so i'm surprised it actually works!